### ISSUE TITLE
Download Code task fails on Windows

### ISSUE THREAD

**Issue by chrdunne** [xebialabs-community/xlr-bitbucket-plugin#5]
Hi 

I am running on Windows 8. I have Cygwin installed.

I have set up a Release, including the Download Code Task.

When the release is started and the task executed, the file extract.sh is created but the code is not downloaded from Bitbucket.

I can successfully open a Cygwin terminal and execute the script and the code will be downloaded.

If I look at the task after execution I see the following in the Output field

'rm' is not recognized as an internal or external command, operable program or batch file. 'rm' is not recognized as an internal or external command, operable program or batch file.

This looks like the message you get when trying to execute rm at a windows DOS command shell. But the extract.sh is obviously intended to be execute in a bash shell.

Do I need to tell XL Release or the plugin to use a particular shell?

Thanks
Chris

**Comment by chrdunne** [comment]
Hi

I found that Cygwin was not on my Windows path. I added it, restarted XL Release, and tried again to execute the task in the release.

This time nothing happens, and nothing is downloaded to the folder. I see no errors in the logs. Also there is no extract.sh file but I assume that this is because it is generated, executed, and then deleted by the task since it can now find the 'rm' command.

If I manually run the steps in the extract.sh script, then it works fine.

Thanks
Chris
